knudev-security/src/main/java/ua/knu/knudev/knudevsecurity/service/AuthenticationService.java

```diff
@@ -21,19 +25,36 @@ public class AuthenticationService implements AuthServiceApi {
 
     @Override
     public AuthenticationResponse authenticate(AuthenticationRequest authReq) {
-        AccountAuth account = accountAuthMapper.toDomain(accountService.findByEmail(authReq.email()));
+        AccountAuthDto account = accountService.findByEmail(authReq.email());
 
-        accountService.checkAccountValidity(account, account.getEmail());
+        checkAccountValidity(account, authReq.email());
 
         authenticationManager.authenticate(
                 new UsernamePasswordAuthenticationToken(authReq.email(), authReq.password())
         );
 
-        Tokens tokens = jwtService.generateTokens(account);
+        Tokens tokens = jwtService.generateTokens(accountAuthMapper.toDomain(account));
```

Comment:
mapper in new line, its good practise to call 1 method in 1 line because it easy to debag if we have errors

Reply:
account domain